components/brave_wallet/browser/asset_ratio_service.cc

```diff
@@ -109,6 +109,18 @@ absl::optional<std::string> ChainIdToStripeChainId(
   return chain_id_lookup->at(chain_id);
 }
 
+base::flat_map<std::string, std::string> MakeBraveServicesKeyHeader() {
```

Comment:
cc @onyb on this since I think he implemented a similar function for the transaction simulation work. Would be good to consolidate this here if possible.